### PR TITLE
Build cache options

### PR DESCRIPTION
**Pull request by alainbodiguel**

### CODE REVIEW

File: pesto-cli/docs/pesto_build.md [hunk 1]
```diff
@@ -105,3 +105,20 @@ The following files are supported by the cascading profiles rule :
     - config.json
     - requirements.json
     - version.json
+
+
+## Note on cache
+By default, the generated `Dockerfile` uses no cache.  
+The main steps are:
```
Comment (sylvaingaudan) [inline pesto-cli/docs/pesto_build.md]: "main steps" of what?
-> of building the docker image - to be specified.

File: pesto-cli/pesto/cli/app.py [hunk 2]
```diff
@@ -45,12 +45,13 @@ def init(target: str,
 @app.command()
 def build(build_config: str,
           profile: List[str]=typer.Option(list(),"--profile","-p",help="Select specific files to update"),
+          cache: str=typer.Option(None,"--cache","-c",help="Select a cache policy (CACHE_UP_TO_PESTO or C1 or CACHE_UP_TO_PIP_REQ or C2 or CACHE_UP_TO_ALGO or C3)"),
```
Comment (sylvaingaudan) [inline pesto-cli/pesto/cli/app.py]: Select a cache policy : CACHE_UP_TO_PESTO (or C1), CACHE_UP_TO_PIP_REQ (or C2), CACHE_UP_TO_ALGO (or C3)

File: pesto-cli/docs/pesto_build.md (hunk 1 above)
Comment (alainbodiguel) [reply pesto-cli/docs/pesto_build.md]: fixed

File: pesto-cli/pesto/cli/app.py (hunk 2 above)
Comment (alainbodiguel) [reply pesto-cli/pesto/cli/app.py]: fixed

**Review by sylvaingaudan — APPROVED** [review]

**Pull request merged**